scripts/BuildAndTest.ps1

```diff
@@ -53,9 +53,9 @@ function Exit-WithFailureMessage($scriptName, $message) {
 
 if (-not $NoBuild) {
     Write-Information "Reformatting code..."
-    dotnet format $RepoRoot\src\Microsoft.Security.Utilities.sln
+    dotnet format --verify-no-changes $RepoRoot\src\Microsoft.Security.Utilities.sln --verbosity diagnostic
```

Comment:
Oh wow, it was already doing this? Was something causing it to no-op because I've never noticed it having anything impact.

Reply:
Nevermind, was looking at individual commit.